test/cmdlineTests/standard_optimizer_generatedSources/output.json

```diff
@@ -35,5 +35,5 @@
         tail := add(headStart, 32)
         mstore(headStart, value0)
```

Comment:
Not really related to this PR, but: Why does github display this in full red :-)? I hope it's them not correctly parsing json - or are the linebreaks we output here actually strictly speaking an issue? Also I hope we properly escape quotes here? (I haven't checked)

Reply:
I think it's doing highlighting only on the snippet in the preview rather than the whole file. It does not have enough context to know it's just a part of a string and this part is not valid JSON on its own :)

Reply:
Maybe...